content/post/2023-11-02-BitalinoMuse/index.md

```diff
@@ -0,0 +1,155 @@
+---
+authors:
+- ana-neves
+categories:
+- Tutorials
+- Biosignals
+date: "2023-11-02"
+title: "Record and Synchronize biosignals from Bitalino with EEG from Muse"
+draft: false
+featured: false
+image:
+  caption: ''
+  placement: 0
+subtitle: ""
+summary: "A step-by-step guide on how to stream, record, and synchronize physiological signals recorded by Bitalino and Muse 2 EEG headband"
+tags:
+- Reality Bending Lab
+- ReBeL
+- University of Sussex
+- Psychology
+- Neuropsychology
+- Physiological
+- Psychophysiology
+- Muse 2 EEG
+- Bitalino
+- Plux
+- Psychobit
+- LSL
+- Lab Streaming Layer
+- Synchronize
+- Send triggers
+---
+
+
+## Equipment
+
+The setup used here is a low cost (< 1000€), easy-to-setup solution for comprehensive bodily recordings with a bit of EEG.
+
+### Bitalino
+
+- Record bodily signals (ECG, PPG, RSP, EDA, ...)
+- 420€
+- [Official website](https://www.pluxbiosignals.com/products/psychobit)
+
+![Alt text](images/BItalino.png)
+
+
+
+### Muse 2 EEG Headset
+
+- Record PPG and 4 EEG channels (TP9, TP10, AF7, AF8)
+- 269.99€
+- [Official website](https://eu.choosemuse.com/products/muse-2)
+
+
+![Alt text](images/MUSE.png)
+
+
+
+## Software
+
+The following software is needed to collect, record and analyse physiological data.
+
+### Open Signals
+
+- [x] Used to visualise the signals recorded from BItalino.
+
+1. Download OpenSignals (r)evolution here: https://biosignalsplux.com/index.php/software
+2. Open device manager to access and configure your biosignalsplux device
+![Alt text](images/image.png)
+
+3. Select advise intended to use.
+![Alt text](images/image-1.png)
+
+4. Access the settings by clicking on the biosignalsplux. Select the channel to which the sensors are connected to.
+
+- *add screenshot*
+(A1) – respiration belt
+(A2) – ECG electrodes
+(A3) – Photosensor/ LUX
+(A4) -  Pulse sensor
+
+5.  Go to Open signals settings, then the integration tab and click on the lab streaming layer.
+![Alt text](images/image-3.png)
+
+6. Start recording.
+![Alt text](<images/image-4 .png>)
+
+-*add screenshot of how the signals look
+
+### Python
+
+- [x] Necessary to run the software to stream Muse signals
+
+1. Download here:  https://www.python.org/downloads/
+2. IMPORTANT!!! Make sure to click on 'Add python.exe to PATH' at the bottom.
+![Alt text](images/image-2.png)
+
+**VS Code**: code editor for python.
+
+1. Download here: https://code.visualstudio.com/download
+
+
+### MuseLSL2
+
+- [x] python package to stream, visualise, and record EEG data from the Muse 2 headset.
+
+1. Open VS Code, click New Terminal.
+2. On the Terminal, write `pip install https://github.com/DominiqueMakowski/MuseLSL2/zipball/main` and press enter.
```

Comment:
@AnafNeves Actually we are not using muselsl but [MuseLSL2](https://github.com/DominiqueMakowski/MuseLSL2) no<, could you update the images to match this